scripts/build-catalogs.mjs

```
@@ -39,6 +39,12 @@ const generateCatalogs = () => {
     ...getVersionArguments(),
   ];
 
+  // optional corporate mirror(s); leave unset for the default (Maven Central + Red Hat GA inferred)
+  const EXTRA_REPOS = process.env.CATALOG_EXTRA_REPOS; // e.g. "https://nexus.corp/repo"
```

ESLint

[error] 'process' is not defined.

(no-undef)